src/main/java/org/amahi/anywhere/service/AudioService.java

```diff
@@ -316,17 +316,17 @@ public void onAudioControlChange(AudioControlChangeEvent event) {
 
     @Subscribe
     public void onAudioControlNext(AudioControlNextEvent event) {
-        startChangedAudio(getNextAudioPosition());
+        BusProvider.getBus().post(new AudioControlChangeEvent(getNextAudioPosition()));
     }
 
     @Subscribe
     public void onAudioControlPrev(AudioControlPreviousEvent event) {
-        startChangedAudio(getPreviousAudioPosition());
+        BusProvider.getBus().post(new AudioControlChangeEvent(getPreviousAudioPosition()));
```

Comment:
similar to[ line#319](https://github.com/amahi/android/pull/358/commits/98bd6cdb02e5eabde08d507f2ca68ad623f8fd2a#diff-73396d7ae4339ff7848b0e2e8d1cfdf1R319)